ext/mbstring/mbstring.c

```diff
@@ -3382,41 +3401,62 @@ PHP_FUNCTION(mb_convert_kana)
 	RETVAL_STR(jp_kana_convert(str, enc, opt));
 }
 
-static int mb_recursive_encoder_detector_feed(mbfl_encoding_detector *identd, zval *var, bool *recursion_error) /* {{{ */
+static int mb_recursive_count_strings(zval *var)
```

Comment:
Just drive by, but I suppose this could be converted to return an unsigned int, as it's either 0 or positive. But that could be done as a follow-up PR.